.travis/install_gradle.sh

```diff
@@ -1,10 +1,8 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
 
 pushd /tmp
 
-wget http://services.gradle.org/distributions/gradle-2.4-all.zip
+wget $(wget -qO- https://services.gradle.org/versions/current | jq -r .downloadUrl)
```

Comment:
As another thought, many Gradle project use the Gradle wrapper anyway, which specifies the exact Gradle version to use. So the version of Gradle used here would only apply to those Gradle projects that do not use the wrapper.